### DISCUSSION TITLE
Ignore files or packages when searching globally for a symbol

### DISCUSSION THREAD

**Discussion by filintodelgado** [golang/vscode-go#3501] [Discussions]
In my daily work I have to deal with repositories that contains vendor packages and it is not uncommon to have the same variable or function in our code and in those vendor packages.

For some, sometimes reason when searching for a **text** using Visual Studio Code global search (`CTRL + SHIFT + F`) the vendor package results might be displayed first which is not only annoying but prevents from finding the actual result that I need.

I solution I found was to exclude those files using `files.exclude` and `search.exclude` features.

Go Extension doesn't respect those options and doesn't have any option of their own when searching for a symbol globally (`CTRL + T`).

Would be nice to have the feature to exclude certain files when searching or the extension could use the values from `search.exclude`.

**Comment by kyegupov** [comment]
Any update here?
It's terribly annoying, Ctrl-T keeps showing me tons of irrelevant symbols from vendored packages.
